src/ethereum_spec_tools/evm_tools/t8n/t8n_types.py

```diff
@@ -417,6 +419,14 @@ def add_transaction(self, tx: Any) -> None:
         else:
             self.successful_txs.append(tx)
 
+    def add_receipt(self, tx: Any) -> None:
+        """
+        Add t8n receipt info for valid tx
+        """
+        txhash = keccak256(rlp.encode(tx))
+        data = {"transactionHash": "0x" + txhash.hex(), "gasUsed": "0x0"}
```

Comment:
If you also want to have the `block_hash` in the receipt, you could add a dummy hash like `bytes.fromhex("00" * 32)`